src/Conf.ml

```diff
@@ -831,6 +831,41 @@ let output =
       & opt (some string) default
       & info ["o"; "output"] ~doc ~docs ~docv)
 
+let ocp_indent_options =
+  [ ("base", None)
+  ; ("type", None)
+  ; ("in", None)
+  ; ("with", None)
+  ; ("match_clause", None)
+  ; ("ppx_stritem_ext", None)
+  ; ("max_indent", None)
+  ; ("strict_with", None)
+  ; ("strict_else", None)
+  ; ("strict_comments", None)
+  ; ("align_ops", None)
+  ; ("align_params", None) ]
+
+let ocp_indent_config =
+  let doc =
+    let open Format in
+    let unsupported =
+      let l =
+        List.filter_map ocp_indent_options ~f:(fun (s, o) ->
+            Option.value_map o ~default:(Some s) ~f:(fun _ -> None) )
+      in
+      if List.is_empty l then ""
+      else
+        asprintf " Options %a are unsupported."
+          (pp_print_list
+             ~pp_sep:(fun fs () -> fprintf fs ",@ ")
+             (fun fs s -> fprintf fs "$(b,%s)" s))
+          l
+    in
+    asprintf "Read .ocp-indent configuration files.%s" unsupported
```

Comment:
My feeling is that it would be more helpful to document the association between ocp-indent options and ocamlformat options, instead of listing the unsupported ocp-indent options.